zeus/api/resources/repository_tests_by_job.py

```diff
@@ -0,0 +1,61 @@
+from flask import current_app
+
+from zeus.config import db
+from zeus.constants import Result, Status
+from zeus.models import Build, Repository, TestCase, Job, Source
+
+from .base_repository import BaseRepositoryResource
+from ..schemas import TestCaseSummarySchema
+
+testcases_schema = TestCaseSummarySchema(many=True, strict=True)
+
+
+class RepositoryTestsByJobResource(BaseRepositoryResource):
+    def get(self, repo: Repository):
```

Comment:
how do I make this less shitty @dcramer ?

Reply:
which part

Reply:
All of it

Reply:
idk looks fine to me, does it work